charts/model-deployment/simple/templates/inferenceservice.yaml

```diff
@@ -11,8 +11,6 @@ metadata:
     autoscaling.knative.dev/target: "1"
     {{- end }}
   name: {{ .Values.name }}
-  finalizers:
-    - inferenceservice.finalizers
```

Comment:
instead of removing it, I'd suggest to add the new finalizers that was introduced.

```
  finalizers:
    - inferenceservice.finalizers
    - odh.inferenceservice.finalizers
```

Can you please update the helm chart in your Gitea environment with both of the finalizers and see if Sync status is bettere Laurent? if it works, we can go with that 😌

Reply:
It works when i add both finalizers.
But IMHO we should alleviate the configuration and let KServe to update both of them automatically.
For ArgoCD, this is not a good practise to play with finalizers that are managed by some software stack automatically (except ArgoCD's own [documented finalizers ](https://argo-cd.readthedocs.io/en/latest/user-guide/app_deletion) of course).